packages/fxa-auth-server/lib/senders/emails/global.scss

```
@@ -61,9 +61,13 @@ $s-10: 40px;
   &-3 div {
     margin-bottom: $s-3 !important;
   }
-
   &-5 {
-    margin-bottom: $s-5 !important;
+    div {
+      margin-bottom: $s-5 !important;
+    }
+    img {
+      margin-bottom: $s-5 !important;
+    }
```

We could do this instead here:
```suggestion
    div, img {
      margin-bottom: $s-5 !important;
    }
```
... but I think we should just leave the extended `mb-5` on that sync logo instead. 🤔